Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,323 @@
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_dtk"};
+};
+class potato_msv_cc: Fic_Soldier_Carbine {// Crew Chief
+  uniform[] = {"rhs_uniform_df15"};
+  vest[] = {"V_TacVest_blk"};
+  headgear[] = {"rhs_zsh7a_mike"};
+  backpack[] = {"B_AssaultPack_rgr"};
+  weapons[] = {SMG};
+  magazines[] = {SMG_MAG,CREW_GRENADES};
+  backpackItems[] = {KEY_EAST,RADIO_LR};
+  items[] = {BASE_MEDICAL,BASE_TOOLS,LEADER_TOOLS,RADIO_MR};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_dtk"};
+};
+class fic_vehicle_crewman: Fic_Soldier_Carbine {// Vehicle crew base
+  headgear[] = {"rhs_tsh4","rhs_tsh4_bala","rhs_tsh4_ess","rhs_tsh4_ess_bala"};
+  magazines[] = {CARBINE_MAG,CREW_GRENADES};
+  backpackItems[] += {KEY_EAST,RADIO_LR};
+  linkedItems[] += {EAST_LEADER_LINKED,BINOS};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_vicl: fic_vehicle_crewman {// Vehicle Leader
+  backpackItems[] = {RADIO_MR,RADIO_LR,KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_vicc: fic_vehicle_crewman {// Vehicle Crewman (non-repair)
+  backpackItems[] = {RADIO_MR,KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_vicd: fic_vehicle_crewman {// Vehicle Crewman (repair)
+  backpackItems[] = {"toolkit",KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_engl: potato_msv_rifleman {// Demoman Leader
+  backpack[] = {"B_Carryall_oli"};
+  items[] += {BASE_ENG,BASE_MEDICAL};
+  backpackItems[] = {"Toolkit",RADIO_MR,KEY_EAST};
+  magazines[] += {BASE_EXP};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_eng: potato_msv_rifleman {// Demoman
+  backpack[] = {"B_Carryall_oli"};
+  items[] += {BASE_ENG,BASE_MEDICAL};
+  backpackItems[] = {"Toolkit",RADIO_MR,KEY_EAST};
+  magazines[] += {BASE_EXP};
+  linkedItems[] = {BASE_LINKED};
+};
+class potato_msv_sf_rifleman: potato_msv_rifleman {// Recon Leader
+  uniform[] = {"rhs_uniform_gorka_r_g","rhs_uniform_gorka_r_y"};
+  vest[] = {"rhs_6b23_6sh116_od","rhsgref_6b23_khaki_rifleman","rhsgref_6b23_khaki"};
+  headgear[] = {"rhs_6b27m_green","rhs_6b27m_green_bala","rhs_6b27m_green_ess","rhs_6b27m_green_ess_bala","rhs_6b28_green","rhs_6b28_green_bala","rhs_6b28_green_ess","rhs_6b28_green_ess_bala","rhs_beanie_green","rhs_beret_milp","rhsusf_Bowman","rhsusf_bowman_cap"};
+  backpack[] = {"rhs_sidor"};
+  handguns[] = {PISTOL};
+  magazines[] += {PISTOL_MAG};
+  attachments[] = {RECONRIFLE_ATTACHMENTS,"rhs_acc_pkas"};
+};
+class potato_msv_sf_ftl: potato_msv_sf_rifleman {// Recon Senior Rifleman
+  vest[] = {"rhs_6b23_6sh116_vog_od"};
+  weapons[] = {GLRIFLE};
+  magazines[] = {GLRIFLE_MAG,GLRIFLE_MAG_HE,GLRIFLE_MAG_SMOKE,LEADER_GRENADES};
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,BINOS};
+};
+class potato_msv_sf_sl: potato_msv_sf_ftl {// Recon Rifleman
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED,RANGE_FINDER};
+  items[] += {RADIO_MR,"rhs_6b27m_digi"};
```

Comment:
I'm going to see what the gear weight looks like across the squad. Maybe we can give the 117 to one of the riflemen or something. I don't fully agree with the need for a 117, but I can see how others would disagree. In my mind scouting would go in one of two directions, long-range scouting, the squad goes out, gathers information, comes back into 148 range and reports, short-range scouting they just report directly over the 148.